src/lib/tradehub/api/APIClient.ts

```diff
@@ -622,6 +624,89 @@ class APIClient {
     return new BigNumber(total).times(poolAllocation).dp(8)
   }
 
+  async getLastClaimedPoolReward(opts: GetLastClaimedPoolRewardOpts): Promise<GetLastClaimedPoolRewardResponse> {
+    const request = this.apiManager.path('markets/get_last_claimed_pool_reward', opts)
+    const response = await request.get()
+    return response.data as GetLastClaimedPoolRewardResponse
+  }
+
+  async getRewardHistory(opts: GetRewardHistoryOpts): Promise<GetRewardHistoryResponse> {
+    const request = this.apiManager.path('markets/get_reward_history', opts)
+    const response = await request.get()
+    return response.data as GetRewardHistoryResponse
+  }
+
+  async estimateUnclaimedRewards(opts: EstimateUnclaimedRewardsOpts): Promise<RestModels.UnclaimedRewards> {
```

Comment:
can you move this to a `LiquidityPoolModule` (`TradeHubSDK.lp.estimateUnclaimedRewards`)